CHANGELOG.md

```
@@ -0,0 +1,55 @@
+### Changed
+
+- Headers type from `map[string]string` to `net/http.Header` (breaking)
+- Path resolution to not encode query parameters
+- Error body reading capped at 1 MiB to prevent memory exhaustion
+- HTTP method is now required (`ErrEmptyMethod` on empty string)
+
+### Fixed
+
+- Lint issues across all source files (errcheck, naming, exhaustruct, etc.)
+
+[Unreleased]: https://github.com/capcom6/go-restkit/compare/v0.1.0...HEAD
```

markdownlint-cli2 (0.22.1)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "unreleased"

(MD053, link-image-reference-definitions)